### PR TITLE
Create test cases for support-scheduler API

### LINKED ISSUE #110: Create Support Scheduler V2 API Test Cases - Positive
Design support-scheduler positive testcases based on [support-scheduler V2 API](https://github.com/edgexfoundry/edgex-go/blob/master/api/openapi/v2/support-scheduler.yaml). Reference: [blackbox-testing support-scheduler.postman_collection](https://github.com/edgexfoundry/blackbox-testing/blob/master/bin/postman-test/collections/support-scheduler.postman_collection.json).

### PR DESCRIPTION
**Pull request by cherrycl**
Create test cases for support-scheduler API

Fix #110 #111 
Signed-off-by: Cherry Wang <cherry@iotechsys.com>

### CODE REVIEW

File: TAF/testScenarios/functionalTest/V2-API/support-scheduler/intervalaction/PATCH-Positive.robot [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+${url}            ${supportSchedulerUrl}
+
+*** Test Cases ***
+IntervalactionPATCH001 - Update intervalaction
+    Given Create Interval And Intervalaction
+    When Update Intervalaction
+    Then Should Return Status Code "207"
+    And Should Return Content-Type "application/json"
+    And Item Index All Should Contain Status Code "200"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
```
Comment (jinlinGuan) [inline TAF/testScenarios/functionalTest/V2-API/support-scheduler/intervalaction/PATCH-Positive.robot:21]:
```suggestion
    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
    And Intervalaction should be updated
```

**Review by jinlinGuan — CHANGES_REQUESTED** [review]
Some testcases need to add `Teardown` to remove interval/intervalaction.

File: TAF/testScenarios/functionalTest/V2-API/support-scheduler/intervalaction/PATCH-Positive.robot (hunk 1 above)
Comment (cherrycl) [reply TAF/testScenarios/functionalTest/V2-API/support-scheduler/intervalaction/PATCH-Positive.robot:21]: Updated.

**Review by jinlinGuan — APPROVED** [review]
LGTM

**Pull request merged**